python/ray/serve/_private/deployment_scheduler.py

```diff
@@ -310,9 +310,7 @@ def on_deployment_deployed(
         assert deployment_id in self._deployments
 
         info = self._deployments[deployment_id]
-        info.actor_resources = Resources.from_ray_resource_dict(
-            replica_config.resource_dict
-        )
+        info.actor_resources = Resources(replica_config.resource_dict)
```

Comment:
This is the bug, `replica_config.resource_dict` is already a flattened dictionary.

Reply:
`Resources.from_ray_resource_dict` is also no longer necessary, because this is the only place it was (incorrectly) being used.